hexrd/ui/overlays/powder_diffraction.py

```diff
@@ -13,7 +13,7 @@ class PowderLineOverlay:
     def __init__(self, plane_data, instr, tvec=np.zeros(3), eta_steps=360):
         self._plane_data = plane_data
         self._instrument = instr
-        self.tvec = tvec
+        self._tvec = tvec
```

Comment:
I had modified this to `self.tvec = tvec` so that it is guaranteed to be converted to a numpy array. `self._tvec = vec` is accepting a list and producing errors in the powder overlay editor.

Can we either:
1. Change it back to `self.tvec = tvec`
2. Do `self._tvec = np.array(tvec)`
?

Reply:
Allowing a list to be passed in is easier, by the way, because it is easier to save via `QSettings` when we save the overlay arguments.

Reply:
solved, keeping option 2, but casting the kwarg to ndarray (with assertion error check)